h2o-algos/src/test/java/water/DKVSnapshotExampleTest.java

```diff
@@ -0,0 +1,40 @@
+package water;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import water.fvec.Frame;
+import water.runner.CloudSize;
+import water.runner.H2ORunner;
+
+@RunWith(H2ORunner.class)
+@CloudSize(1)
+public class DKVSnapshotExampleTest {
+    @Test
+    public void histogramShowcase() {
+        Scope.enter();
+        try {
+            final Cleaner.Histo before = Cleaner.Histo.current(true);
+            final Frame frame = Scope.track(TestUtil.parse_test_file("./smalldata/iris/iris_wheader.csv"));
+            final Cleaner.Histo after = Cleaner.Histo.current(true);
+
+            System.out.println(after._total - before._total);
```

Comment:
Does this output a list or change in number of keys? Was the request to get a list of keys?

Reply:
The snapshot below does that. I added one more sample into that test method.